Mathlib/Topology/Algebra/RestrictedProduct.lean

```diff
@@ -545,13 +666,13 @@ theorem isOpenEmbedding_structureMap :
 include hAopen in
 theorem nhds_eq_map_inclusion {S : Set ι} (hS : cofinite ≤ 𝓟 S)
     (x : Πʳ i, [R i, A i]_[𝓟 S]) :
-    (𝓝 (inclusion R A hS x)) = map (inclusion R A hS) (𝓝 x) := by
+    (𝓝 (inclusion R A hS x)) = (𝓝 x).map (inclusion R A hS) := by
```

Comment:
I don't know if this is written down somewhere, but I think this is another case where we deliberately avoid dot notation. The argument here (apart from the fact that Patrick finds this unreadable) is that we want the notation to be similar to the image of a set under a function, so the function should come first.

`.map` should work, right? Otherwise even `Filter.map` is fine I think.

Reply:
I changed it because after defining RestrictedProduct.map it became ambiguous. What do you suggest?

Reply:
I suggest `.map _ _`, which should resolve to `Filter.map _ _` thanks to the expected type